### ISSUE TITLE
Distribute Beaker Browser via dat and fetch updates from there

### ISSUE THREAD

**Issue by probonopd** [beakerbrowser/beaker#761]
Currently `dat://beakerbrowser.com/docs/install/` links to GitHub Releases for the downloads.

It would be nice to have `dat://` download links instead. Please also distribute Beaker Browser via `dat://beakerbrowser.com/` and fetch updates from there.

**Comment by pfrazee** [comment]
Before we can do this, we'll need to update `electron-updater` to support Dat as a transport.

**Comment by probonopd** [comment]
...or [AppImageUpdate](https://github.com/AppImage/AppImageUpdate), I guess
